uktrade/data-hub-frontend
# Reactify investment project details page

## Description of change

The investment project details page has been reactified.

This PR is very large as I had to add several new fixtures as well as numerous assertions around large tables.

## Test instructions

Go to the details page for an investment project. All information should appear as normal. It should be possible to change the project stage and view incomplete fields as normal.

## Screenshots
  <details>
  <summary>No requirements or value</summary>
  
  ### Before

![Screenshot 2023-08-16 at 13-37-10 Department for Business and Trade](https://github.com/uktrade/data-hub-frontend/assets/36161814/e04c8366-ec9b-4cd2-aa34-f7976e1c4861)


  ### After
  
![Screenshot 2023-08-16 at 13-36-45 Department for Business and Trade](https://github.com/uktrade/data-hub-frontend/assets/36161814/0edb869d-0e70-4a96-a6c5-066c04281736)


  </details>

  <details>
  <summary>All fields filled in</summary>
  
  ### Before

![Screenshot 2023-08-16 at 13-35-47 Department for Business and Trade](https://github.com/uktrade/data-hub-frontend/assets/36161814/86875dbe-7f4d-4b36-a528-bf2e15c5caf4)


  ### After

![Screenshot 2023-08-16 at 13-35-21 Department for Business and Trade](https://github.com/uktrade/data-hub-frontend/assets/36161814/ef37f7a0-401c-461f-b43d-02cf0f79ef77)

  
  </details>

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
